apps/moderatorfeedback/fields.py

```diff
@@ -0,0 +1,117 @@
+import re
+import unicodedata
+
+from django.core import checks
+from django.core.exceptions import FieldError
+from django.db import models
+from django.utils.encoding import force_text
+from django.utils.functional import curry
+from django.utils.safestring import mark_safe
+from django.utils.translation import ugettext_lazy as _
+
+
+class ModeratorFeedbackField(models.CharField):
+    description = _("")
+
+    def __init__(self, *args, **kwargs):
+        kwargs['max_length'] = 254
+        kwargs['null'] = True
+        kwargs['default'] = None
+        kwargs['blank'] = True
+        super(ModeratorFeedbackField, self).__init__(*args, **kwargs)
+
+    def deconstruct(self):
```

Comment:
No need for a custom deconstuct() here, as we use just the default arguments that are removed by the deconstruct of the super class.